src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborDataItem.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+
+namespace System.Security.Cryptography.Encoding.Tests.Cbor
+{
+    internal enum CborMajorType : byte
+    {
+        UnsignedInteger = 0,
+        NegativeInteger = 1,
+        ByteString = 2,
+        Utf8String = 3,
+        Array = 4,
+        Map = 5,
+        Tag = 6,
+        Special = 7,
+    }
+
+    /// Represents the Cbor Data item initial byte structure
+    internal struct CborDataItem
+    {
+        private const byte AdditionalInformationMask = 0b000_11111;
+        public readonly byte InitialByte;
```

Comment:
Generally we don't do public fields, perhaps `public byte InitialByte { get; }` ?